master/internal/api_config_policies_intg_test.go

```diff
@@ -414,3 +544,1893 @@ func setupConfigPolicyAuthZ() *mocks.ConfigPolicyAuthZ {
 	}
 	return cpAuthZ
 }
+
```

Comment:
Excellent test coverage in `TestValidatePoliciesAndWorkloadType`. It's fine as is. I think there's a lot of overlap with test cases for the `configpolicy` unmarshaling functions and we could have fewer test cases here without impacting overall coverage.

For this test function, I would think we need to test:

1. workload type

- invalid
- empty

2. config policies

- empty
- yaml: valid experiment (can make this complex: include special characters, partial config sections, etc)
- yaml: valid ntsc
- yaml: invalid experiment (one error or two error types is fine)
- yaml: invalid ntsc
- json: valid experiment
- json: valid ntsc
- json: invalid experiment
- json: invalid ntsc

Reply:
All covered! 😄